bin/identify_dormant_github_users.py

```diff
@@ -111,18 +69,11 @@ def message_to_slack_channel(list_of_dormant_users: list) -> str:
 def identify_dormant_github_users():
     env = EnvironmentVariables(["GH_ADMIN_TOKEN", "ADMIN_SLACK_TOKEN"])
 
-    dormant_users_from_csv = get_dormant_users_from_github_csv(
-        GithubService(env.get('GH_ADMIN_TOKEN'), ORGANISATION)
-    )
+    dormant_users_from_csv = get_dormant_users_from_data_lake(GithubService(env.get('GH_ADMIN_TOKEN'), ORGANISATION))
```

Comment:
The variable name needs to change. `dormant_users_according_to_github`?

Reply:
ah yeah, missed that